cms/models.py

```diff
@@ -1704,13 +1715,34 @@ class CertificatePage(CourseProgramChildPage):
         help_text="Specify the course/program name.",
     )
 
+    institute_text = models.CharField(
+        max_length=250, null=True, blank=True, help_text="Specify the institute text"
+    )
+
     CEUs = models.CharField(
         max_length=250,
         null=True,
         blank=True,
         help_text="Optional text field for CEU (continuing education unit).",
     )
 
+    partner_logo = models.ForeignKey(
+        Image,
+        null=True,
+        blank=True,
+        on_delete=models.SET_NULL,
+        related_name="+",
+        help_text="Optional Partner logo",
```

Comment:
Ideally, We should mention the required image size too. e.g. 500x500 etc. You can see the examples for other image fields in the cms/models.py.